workouts/workoutsRoutes.js

```diff
@@ -13,4 +13,48 @@ router.get('/:id', async (req, res) => {
   }
 });
 
+router.get('/', checkLoggedIn, async (req, res) => {
+  try {
+    const workouts = await DB.getWorkouts();
+    return res.status(200).json(workouts);
+  } catch (error) {
+    return res.status(500).json({
+      Error: error,
+    });
+  }
+});
+
+router.get('/', async (req, res) => {
+  try {
+    const workouts = await DB.getWorkouts();
+    return res.status(200).json(workouts);
+  } catch (error) {
+    return res.status(500).json({
+      Error: error,
+    });
+  }
+});
```

Comment:
This is a repetition of the code on line 16-25 without the restrictedMiddleware access